docs/PR.md

```diff
@@ -0,0 +1,7 @@
+# Why is this PR needed?
+
+# What does the PR do?
+
+# Does it break backwards compatibility?
+
+# List of future improvements not on this PR
```

Comment:
I like these points!

I would make it less wordy though. Making this too heavy might discourage people from making smaller PRs. For example, most of my PRs don't have breaking changes and don't have future improvements (such as refactors). I probably only want to write a short sentence describing "what" and "why".

How about this? Happy to discuss further!

-----


### Description

**What** does this do and **why**?

### Future improvements

Remove if not relevant

- [ ] Breaking change

-----